### ISSUE TITLE
Duplicate dropdown option (v0.7.1)

### ISSUE THREAD

**Issue by ITMan101** [vassbo/freeshow#115]
### Initail Issue Found
The duplicate dropdown option does not do anything when used on a slide in the middle of a section, in the "show" or "edit" tabs. And when used on the first slide of a section, duplicates the whole section to the end of the show. It also sends you to the "show" tab when used in the "edit" tab.

### How initial issue affected me
I usually would use the duplicate option while editing a song when the automatic search and song creation ends up with a slide which needs to be split. I would duplicate said slide and then remove half of the content from the original and then the other half from the newly created duplicate.

### Other related issues found after exploration
The duplicate option does nothing for duplicating a layout.
When duplicating a Show within a project it creates a whole new Show and adds it to the folder. I was expecting it to work as if I had dragged a new instance of the same Show into the project or had used the "add to show" option multiple times.

### Unaffected
I have tested and duplicating overlays, templates and shows functions as I expected. There was no duplicate option for Projects found.

**Comment by vassbo** [comment]
Thank you. I have actually just fixed some of these issues in the dev build, and will see if I can fix all of them for the next version.

**Comment by ITMan101** [comment]
Awesome to hear!

**Comment by ITMan101** [comment]
Something I noticed with the v0.7.2 update is when you duplicate a slide it places it all the way at the end of the Show and places it in a new Group. It would be nice if it would place the slide right after the slide you duplicated and in the same Group.

**Comment by vassbo** [comment]
Thanks for the feedback. I will see what I can do.

**Comment by ITMan101** [comment]
I believe this was working in a previous version. As of v0.8.2 when duplicating a slide it adds it to a new group instead of to the group of the slide you are duplicating.

**Comment by vassbo** [comment]
Okay, just to clarify. If you duplicate the "parent" slide you should get a new group, but if you duplicate one of the "child" slides it should add the new slide to the same parent, instead of creating a new group. I will fix this!

**Comment by ITMan101** [comment]
Yes, duplicating a "child" slide currently creates a new group as of v0.8.2. However, I feel like there should be a way to duplicate the "parent" slide without duplicating the whole group.

**Comment by ITMan101** [comment]
Sorry, didn't mean to close this 😅

**Comment by vassbo** [comment]
On the right side under "Groups" you have access to all of the groups. Here you can click or drag them into the current show.

**Comment by ITMan101** [comment]
Perhaps the "duplicate" option should be only for slide-based duplication and group-based duplication should be left to the "Groups" menu on the side? Right now you cannot duplicate a "parent" slide within the show.

**Comment by ITMan101** [comment]
Currently, the duplicate dropdown option isn't consistent across its implementations. When clicking on the "parent" slide it does one thing, and when clicking on a child slide it does another. It might make for an easier learning curve and a more understandable user interface if it does the same thing no matter if it is a "parent" or "child" slide.

**Comment by vassbo** [comment]
Agreed!
